jsonrpc/jsonrpc.go

```diff
@@ -133,8 +133,8 @@ func (j *JSONRPC) setupHTTP() error {
 	}
 
 	if j.config.TLSCertFile != "" && j.config.TLSKeyFile != "" {
-		j.logger.Info("https cert file", j.config.TLSCertFile)
-		j.logger.Info("https key file", j.config.TLSKeyFile)
+		j.logger.Info("TLS", "cert file", j.config.TLSCertFile)
+		j.logger.Info("TLS", "key file", j.config.TLSKeyFile)
```

Comment:
The same question applies?

Reply:
Yes, because that way we can see what files are exactly used as certificate files. It is important for troubleshooting, if something is not working we can quickly check what files are given as certificate files.